### PR TITLE
[WebProfilerBundle] Add a title and img role to svg of the web debug toolbar

### PR DESCRIPTION
**Pull request by Saladeuh**
| Q             | A
| ------------- | ---
| Branch?       | 6.3 
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT

This changes improves the accessibility for blind/low vision developpers. This have not visible effect for not impaired developpers.

### CODE REVIEW

**Comment by javiereguiluz** [comment]
@Saladeuh thanks for this. It's a good idea to improve accessibility like this.

The problem is that in this PR you've removed all the new icons that we added in the profiler redesign (see #47148) and you used the legacy icons. So, please update this PR to add the `<title>` to the new icons. Thanks.

**Comment by Saladeuh** [comment]
Thank @javiereguiluz ! I'm sorry, I modified the icons from a 5.4 project. It would be better now.

File: src/Symfony/Bundle/WebProfilerBundle/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.3
+---
+
+* Add a role=img and a explicit title in the .svg using in the toolbar to improve accessibility with screen readers for blind users
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/WebProfilerBundle/CHANGELOG.md]: missing space at start of line

**Review by nicolas-grekas — APPROVED** [review]

**Review by javiereguiluz — APPROVED** [review]

File: src/Symfony/Bundle/SecurityBundle/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * Deprecate enabling bundle and not configuring it
  * Add `_stateless` attribute to the request when firewall is stateless
+ * Modify the icon.svg to improve accesssibility for blind/low vision developpers
```
Comment (ro0NL) [inline src/Symfony/Bundle/SecurityBundle/CHANGELOG.md]: `accessibility` and `developers`

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @Saladeuh.

**Pull request merged**